src/util.cpp

```diff
@@ -8,18 +8,21 @@
  * @copyright Copyright (c) 2023
  * 
  */
+
 #include "util.hpp"
 
 #include "cli/conio.hpp"
 #include "consts.hpp"
 
 #include "Particle.h"
 
+/*!
+    \brief A macro that stores the the size of each line of output as a constant 16 bytes
+*/
```

Comment:
In general, we prefer using the Javadoc style (as seen in other parts of the code), however, this will be acceptable for now.

Reply:
Javadoc style would look like this, correct?:

/**
 * \brief A macro that stores the the size of each line of output as a constant 16 bytes
 */

Looking at the html doc, it looks like there isn't a difference in output between using /** and /*!, so I kept /** to better match the other comments in the code.

Reply:
Correct,

```
/**
 *
 */
```
and
```
/*!
 *
 */
```
are functionally identical.

Reply:
Thanks for the clarification! (also committed again)